css/index.css

```diff
@@ -79,21 +88,40 @@ padding: 0;
 }
 
 .experience-item {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color:rgb(178, 240, 240);
+    color: rgb(104, 58, 164);
+    border-radius: 1.5em;
+    padding: 0.5em;
+}
+.exp-description {
     display: flex;
     flex-direction: row;
     justify-content: space-between;
     align-items: center;
-    background-color:rgb(223, 236, 236);
-    color: rgb(54, 22, 96);
-    border-radius: 0.8em;
+    border-radius: 1em;
     padding: 0.5em;
 }
 
 .date {
+    display: flex;
+    flex-direction: column;
     font-weight: bold;
+    margin: 1em;
     min-width: fit-content;
     }
 
+.grid {
```

Comment:
Could you explain the idea behind this style? I think it would be clearer with some skills added to your portfolio. And the same is in your Projects section